Tutorials/Creating-and-distributing-a-package/index.md

```diff
@@ -0,0 +1,500 @@
+
+In short - similar options to what you can set on the package upload section on Our when uploading a package:
+
+![Our package data][our-pkg-data]
+
+So before we can try this out, let's go to Our and create an API key!
+
+### Creating an Our API key
+
+If you had back to Our Umbraco and visit the [package overview][member-package-overview], then you will notice that there is a button under your package that says "API Keys". 
+Each package api key is tied to that specific package, if you visit the page then you can create keys with a name you can use to differentiate the keys. Once you make a key it will show once, as soon as you refresh or navigate away the key will be gone and you'll have to make a new one if you lose it.
+
+Once you created your key, make sure to copy and paste it somewhere. We will need to use it a few times in the coming steps, remember if you lose it you will have to make a new one!
+
+### Pushing to Our with UmbPack
+
+Now that we have an API key we can try to push our package update to Our Umbraco. This can be done like this:
+
+```umbpack push .\PackageWorkshopDashboard_1.0.0.zip -k [Api key here]```
+
+:::note 
+If a package with the same name already exists it may give an error. In that case you can run `umbpack pack .\package.xml -v 1.0.1` to create a new version of the package then push it with the above command after editing the path to the new package.
+:::
+
+An important thing to note with the push command here is that it sets some defalt values. If not specified otherwise it will default to saying your package is compatible with Umbraco v8.5.0, Dotnet 4.7.2 and it's to be set as the new "current" package on Our.
+
+You can edit all of these defaults, and also specify older versions of your package to be archived when pushing new versions. You can read much more about the push options in the [Umbpack documentation][umbpack-push].
+
+So at this point we can work on our package locally, build a new version within seconds by running the pack command and then deploy it to Our using the push command!
+
+Not easy enough for you? Let's try automating this entire thing with Github actions then!
+
+## Deploy your package using Github Actions
+
+If you think back to the beginning when we set up our sites using the Package Templates you may remember that by default you get a Github action installed as well.
+
+If you check out the `~/.github/workflows` folder in your solution, you will see there is a readme file and a build.yml file. 
+
+The build.yml file is used by Github actions, which will perform some tasks for you when certain criteria are met. If you never worked with continuous integration and deployment (CI/CD) before, then this may seem like magic - but don't worry we will run through the commands!
```

Comment:
```suggestion
The build.yml file is used by Github actions, which will perform some tasks for you when certain criteria are met. If you haven't worked with continuous integration and deployment (CI/CD) before, then this may seem like magic - but don't worry we will run through the commands!
```